site.yml

```diff
@@ -53,7 +55,9 @@
     - role: inspec
       install_inspec_controls: [common]
       tags: [common,inspec]
-      when: inspec.enabled|default("False")|bool
+      when:
+        - inspec.enabled|default("False")|bool
+        - ursula_os == 'rhel'
```

Comment:
This common control is in 'ursula-inspec': https://github.com/blueboxgroup/ursula-inspec/
and it's about password 90 days.
for serverspec's common role os_spec.rb (https://github.com/blueboxgroup/ursula/blob/master/roles/common/templates/etc/serverspec/os_spec.rb), it contain password 90 days too.
I modify the condition here is to avoid the duplicate check.
when all ubuntu serverspec move to inspec, we can remove this condition accordingly.